sklearn/preprocessing/imputation.py

```diff
@@ -8,13 +8,14 @@
 from scipy import sparse
 from scipy import stats
 
+from ..neighbors import KDTree, NearestNeighbors
```

Comment:
Yes. Test runs perfectly on my mac.. I can remove those lines and make another commit to try.